### DISCUSSION TITLE
New user of CWatM

### DISCUSSION THREAD

**Discussion by PeterBurek** [iiasa/CWatM#36] [General]
Dear new user of CWatM,

we have a growing community of users, which is great.
Unfortunately we do not always have the time to support you, therefore we are trying
to create something where users help users.

We provide the model and the data, but we are loosing track who is using CWatM. 
It would be nice, if you can introduce yourself here, and maybe write why and for which reason and area you want to use CWatM
Thanks

Peter

**Comment by djmitra123** [comment]
Hi,
I am Dev. I am using the CWatM model with my own data to learn more about the Global hydrological models. The goal is essentially to check the impact on groundwater systems around the globe.

**Reply by djmitra123** [reply]
@CWatM, 
I accessed the data using the ftp link that you sent me. However, I can neither open the text files nor download the data in the local directory. Can I use them directly using the ftp link?

**Reply by Han-Su22** [reply]
Hi Dev,

How did you open the FTP link? I open the FTP link with the Windows File Explorer, and then I could copy all the files and paste them into the local folder. It works for me.

**Reply by djmitra123** [reply]
Hi Han,
I did the same in windows but could not copy it. However, it worked on Mac. So I have downloaded the data. I was also able to run the rhine catchment example which was surprisingly very quick.

**Reply by Zprivf** [reply]
Hi, My name is Fitsume,
I am using CWatM for water availability study.
Ubuntu on windows should work fine for downloading from windows. Hope this helps.

**Reply by djmitra123** [reply]
Hi @Han-Su22 @Zprivf,
Thank you for your responses! Quick question: were you able to download and unzip the cwatm_input5min.zip folder?

**Reply by Han-Su22** [reply]
Yes, I could. This file is too large, 13GB. I downloaded it several times and finally succeeded. It may break during the downloading.

**Reply by djmitra123** [reply]
Oh I see. For me, it only shows 1.25 gb in a zipped folder. I guess the data is not complete in my case. Can you help me with this dataset? I would love to run the 5min global simulation. Will you be able to share this dataset temporarily or @CWatM  if you can fix the data.
Thanks

**Reply by Han-Su22** [reply]
Hi @djmitra123. I could but I suggest you have another try during the night or weekend of Vienna time. If it still doesn't work, I could try to share it. Good luck.

**Reply by djmitra123** [reply]
Sounds good! I will give it a shot during the weekend. :)

**Reply by emmy-code** [reply]
Hi, @djmitra123 It's nice to see that you are exploring and making some headway with the model.
I tried installing the model after installing all the dependencies but I was getting an error message. Please can you provide some advice for the installation?

**installation command**
pip install git+git://github.com/iiasa/CWatM

<img width="670" alt="image" src="https://user-images.githubusercontent.com/66244085/169344667-2f1c6843-063f-43dc-ba82-8091ca8e9be8.png">

Thank you

**Reply by Han-Su22** [reply]
Hi Emmanuel,

You don't need to install CWatM via pip. You can simply download the code on Github and follow the tutorials to run the code/model.

**Reply by emmy-code** [reply]
Hi Han,

Thank you for the response.
I will do just that and I hope it works. Really appreciate the advise.

**Comment by Han-Su22** [comment]
Hi Peter and everyone,

I am Han, a PhD candidate from University of Twente. I am trying to evaluate agricultural water consumption with CWatM and crop models, e.g. WOFOST. Compared to other models, CWatM is relatively easier to start and python friendly. The code of CWtaM is also clean and tidy. So, I see the flexibility of CWatM to couple with other models. Hope we all have fun with CWatM.

**Reply by emmy-code** [reply]
Thank you @Han-Su22 for your comment, I am trying to use the CWatM for Hydrological modelling for my Ph.D research. Do you mind if we can collaborate for exchange of ideas?

Kind regards.

**Reply by Han-Su22** [reply]
Good idea, but I am also not kind of expert. Exchanging of ideas sounds good. Shall we discuss via email to see what we can do? My email is h.su@utwente.nl

**Reply by emmy-code** [reply]
Thanks for the response. I have sent you an email.

**Comment by Supsa** [comment]
Hi,
I am Alexandre, postdoc from Univ Rennes. I have been using CWatM for few months so far. I intend to study the impact of different land management (artificialization...) or soil properties on water ressource in several watersheds located in Brittany, France.
Thanks a lot for developping such a model, it is a considerable help for many research questions!

**Comment by emmy-code** [comment]
Hi Peter and Everyone. 
My name is Emmanuel, I am a PhD candidate with West African Science Service Centre on Climate Change and Adapted Land Use (WASCAL), I recently came across the CWatM model and it seems great. I hope to explore the model in answering some research questions for my study area in Nigeria.

**Reply by mikhailsmilovic** [reply]
Welcome, Emmanuel 💧
Looking forward to seeing CWatM applied to Nigeria!

**Comment by JeromeJeitany** [comment]
Greetings,
My name is Jerome and I am currently a PhD Candidate at the University of Florence.  The work intends on exploring the nexus of water energy food and ecosystem, where CWAtM provides a suitable tool to address the water component. I would like to inquire if its possible to for a higher resolution of the data while using own personal data (in .nc format). 

Kind Regards,

**Reply by mikhailsmilovic** [reply]
Welcome, Jerome 💧
It is possible to run at higher resolutions!
CWatM has been run at 30 arc minutes, 5 arc minutes, 30 arc seconds, and 1 square kilometre.

**Comment by HydroVadim** [comment]
Hi Peter and everyone,
My name is Vadim and I work as a junior researcher  at the Lomonosov Moscow state university, land hydrology department. I am interested in using CWatM for research and educational purposes. My work is related to water balance issues, primarily river flow and water storage. Here is my ORCID page if anyone is curious. https://orcid.org/0000-0001-8757-7344.
At the moment I'm going through your wonderful tutorials, number 4 in particular, that requires the gloabl data set from your serve. Unfortunately for some reason I am unable to send an email with the request to wfas.info@iiasa.ac.at. I understand that I can obtain simillar datasets from other sources, but during the learning I would like to keep possible sources of error to a minimum, including dataset structure, metadata etc. Could you please give me link to the dataset?

**Reply by Han-Su22** [reply]
Hi Vadim,

Did you try the ftp link in tutorial/exercise 5? It works for me.

**Reply by HydroVadim** [reply]
Hi Han-Su,
Surprisingly no. I have tried several links in tutorials but they don't open any specific site, just browser starting page. To be honest, I probably have some issues with browser settings since any browser tries to open the link by Opera.

**Reply by HydroVadim** [reply]
and Opera is not even my default browser

**Reply by Han-Su22** [reply]
Can you try to copy the FTP link and past it to your File Explorer (as the way you open the local files)?

**Reply by HydroVadim** [reply]
Ok, I get it. The browsers can not open the links - https://serverfault.com/questions/1064804/ftp-link-opens-to-blank-page.
But it works fine by WinSCP. Thank you for support)

**Reply by HydroVadim** [reply]
Yes! It works as well. I didn't know it can be done this way.

**Comment by behvr1** [comment]
Hello everyone,
My name is Xinchen and I am currently a PhD student at Tianjin University and the China Institute of Water Resources and Hydropower Research. I have just started to get into CWatM model and I want to use CWatM model for my research in Central Asia (which is also my PhD thesis). I hope to learn the model and start my research. 
I am new to this model. I hope to discuss any problems in learning the model. My email is: xinchengu@foxmail.com.

**Comment by Chengwenju** [comment]
hello everyone
My name is Wenju Cheng, and I am currently a PhD Candidate at Northwest Institute of Eco-Environment and Resources, Chinese Academy of Sciences.  I'm still a novice in CWatM model. I want to use the CWatM model to couple a glacier hydrological model to more accurately describe the changes in groundwater recharge and storage in northwest China. But so far, I don't know how to couple the two models. If there are researchers interested in this work, we can communicate with each other. My email: chengwj@lzb.ac.cn

**Comment by Abicula** [comment]
Hi Everyone.
My name is Abdullah Alyamani, and I am a master's student in King Abdullah University of Science and Technology. I received my undergraduate in Boston University as an Earth and Environmental Science, and currently I am pursuing master's as Earth Science and Engineering. 
I am trying to learn the CWatM, get comfortable with it and its code in order to adapt it to understand groundwater budget in Saudi Arabia, but baby steps are important.
I am following the model tutorials now.

**Reply by dof1985** [reply]
Hello Abdullah

It is great to have you join the CWatM community. I would like to suggest you also read the following paper on CWatM-Modflow coupling, which would be important for GW hydrologic simulations: https://gmd.copernicus.org/articles/15/7099/2022/

**Comment by abbylute** [comment]
Hi All,

My name is Abby Lute. I am a postdoctoral researcher at the Woodwell Climate Research Center in the US. I am hoping to use CWatM to provide hydrological contributions to the climate risk assessments my team produces for countries and municipalities. I look forward to learning to use the model.

**Comment by abhishekdalwi** [comment]
Hello Peter and everyone.

I am Abhishek Dalwi, and I am a Project Associate at the National Environmental Engineering Research Institute (CSIR-NEERI), India. I intend to use CWatM for hydrological modelling research in my study area. Thanks for developing and maintaining CWatM.

**Comment by umarm2608** [comment]
Hello Peter and everyone,

I am Umar, working as a Research Associate. We want CWatM outputs for the Indus Basin after calibrating on the Indus. The Indus River is the lifeline of Pakistan.

I’m wondering about the discharge_totalavg and baseflow_totalavg files. They are in PathOut, but the data I downloaded from Google Drive (cwatm_input_5min) doesn’t have any folder containing these two files. Does anyone know why this is?

<img width="814" height="326" alt="image" src="https://github.com/user-attachments/assets/a66daec9-dadf-4cb5-9c71-c5ac2ea799ac" />
